mysql-test/suite/innodb/r/monitor.result

```diff
@@ -661,6 +660,16 @@ SELECT NAME, COUNT > 0 FROM INFORMATION_SCHEMA.INNODB_METRICS WHERE NAME
 LIKE 'buffer_page_written_index_leaf';
 NAME	COUNT > 0
 buffer_page_written_index_leaf	1
+SET GLOBAL innodb_monitor_disable=module_purge;
+SET GLOBAL innodb_monitor_disable=module_log;
+SET GLOBAL innodb_monitor_disable=module_icp;
+SET GLOBAL innodb_monitor_disable=module_lock;
+SET GLOBAL innodb_monitor_disable=module_compress;
+SET GLOBAL innodb_monitor_disable=module_index;
+SET GLOBAL innodb_monitor_disable=module_adaptive_hash;
+SET GLOBAL innodb_monitor_disable=module_file;
+SET GLOBAL innodb_monitor_disable=module_ibuf_system;
+SET GLOBAL innodb_monitor_disable=module_innodb;
```

Comment:
It would be nice to have a test that shows which counters are part of which module.

Furthermore, I think that we should actually test that each counter does get incremented. That may be unreasonable for some counters of `module_ibuf_system` and `module_adaptive_hash`, but should be doable with small test cases for the other counters.